### ISSUE TITLE
Contact Form Not Passing Field Info to Email Name and Subject

### ISSUE THREAD

**Issue by gallnomad** [elementor/elementor#11000]
I looked for duplicate issues but could not find any and after contacting Pro support they told me to add a request here. 

## Prerequisites

- [x ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
When the contact form in elementary is passed to the inbox. All of the information submitted by the customer is passed into the body of the email and not in the heading information. So all emails from the form that hit your inbox look exactly the same until you open them. This makes it extremely difficult to deal with customer requests and followups. I believe this is basic functionality that should be included in Elementor, though your support team told me it was not and not being changed anytime soon. I don't want to have to use another contact form when I have paid for elementor.

**Describe the solution you'd like**
Have the fields the customer inputs be passed to the header information in the email.

**Describe alternatives you've considered**
I could use another companies contact form but I don't want too seeing as I have paid for the highest level pro version and as I say this should be basic functionality. I have not seen this issue with other form builders or even when I used opencart previously.

**Additional context**
I'm adding two screen shots. One where the information is being pulled from. I tried to add shortcode here but got errors. 

![test](https://user-images.githubusercontent.com/47094628/77838731-b397d680-71a0-11ea-9964-ef6b8eaebedd.png)

The second image is how the information appears in email and as you can appreciate when you have 20 or more customer requests at one time in your inbox, the lack of subject and email name from the customer make it very hard to monitor and control effectively.

![Selection_002](https://user-images.githubusercontent.com/47094628/77838737-c0b4c580-71a0-11ea-89b4-7af5e17865a3.png)

**Comment by pixelterritory** [comment]
This would be very useful indeed. 👍

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.9.3

Feel free to update

Thanks!

**Comment by gallnomad** [comment]
Hi,

Sorry to be the bearer of bad news but it has not been fixed. When you look at your inbox all emails sent via the contact form come in from your domain not the customers email. I have not tested the subject line as I had to remove it from my site, regardless emails should be coming from the customer not my domain name.

I just updated, cleared cache and even incognito to be sure and the issue still exists.

Cheers
